src/test/java/spring/boot/com/sbparser/ReviewMapperTest.java

```diff
@@ -0,0 +1,95 @@
+package spring.boot.com.sbparser;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.util.Set;
+import org.junit.Assert;
+import org.junit.Test;
+import spring.boot.com.sbparser.dto.ReviewDto;
+import spring.boot.com.sbparser.mapper.ProductMapper;
+import spring.boot.com.sbparser.mapper.ReviewMapper;
+import spring.boot.com.sbparser.mapper.UserMapper;
+import spring.boot.com.sbparser.model.Product;
+import spring.boot.com.sbparser.model.Review;
+import spring.boot.com.sbparser.model.Role;
+import spring.boot.com.sbparser.model.User;
+
+public class ReviewMapperTest {
+    private static final String DEFAULT_PASSWORD = "1111";
+    private static final String DEFAULT_ROLENAME = "USER";
+    private static final String PRODUCT_ID = "B001GVISJM";
+    private static UserMapper userMapper = new UserMapper();
+    private static ProductMapper productMapper = new ProductMapper();
+    private static ReviewMapper reviewMapper = new ReviewMapper(productMapper, userMapper);
```

Comment:
we can mock the work of userMapper and productMapper - so that we could test only the work of reviewMapper in this class

Reply:
done